.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .auth.json
 _cache.db
 all*.json
+config.json
```

Comment:
Should be in another PR, but got lazy. Didn't want to include my local `config.json` that was already in my local clone.